plugin.video.dailymotion_com/default.py

```diff
@@ -265,46 +258,34 @@ def searchUser():
         searchl_string = keyboard.getText().replace(" ", "+")
         listUsers(urlMain+"/users?fields=username,avatar_large_url,videos_total,views_total&search="+searchl_string+"&limit="+itemsPerPage+"&family_filter="+familyFilter+"&localization="+language+"&page=1")
 
-def playVideo(id,live=False):
+def playVideo(vid,live=False):
     if live:
-        url=getStreamUrl(id,live=True)
+        url=getStreamUrl(vid,live=True)
     else:
-        url = getStreamUrl(id)
+        url = getStreamUrl(vid)
     #xbmc.log("DAILYMOTION - url = %s" %url,xbmc.LOGNOTICE)
     if url:
         listitem = xbmcgui.ListItem(path=url)
         xbmcplugin.setResolvedUrl(pluginhandle, True, listitem)
     else:
         print 'No playable url found'
```

Comment:
Please update the print statements to the print function.
```suggestion
         print('No playable url found')
```

Reply:
or even better replace them with `xbmc.log(...)`